<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DamageManager</class>
 <widget class="QDialog" name="DamageManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>350</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>69</red>
        <green>78</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>57</red>
        <green>65</green>
        <blue>67</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>34</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>69</red>
        <green>78</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>57</red>
        <green>65</green>
        <blue>67</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>34</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>69</red>
        <green>78</green>
        <blue>81</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>57</red>
        <green>65</green>
        <blue>67</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>34</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>23</red>
        <green>26</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>46</red>
        <green>52</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Health Manager</string>
  </property>
  <widget class="QSpinBox" name="spin_raw_health">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_raw_health">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>10</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_bonus_health">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>40</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_bonus_health">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_raw_max">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Raw Max Health</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bonus_health">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bonus Health</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rst_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>320</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_health">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Temporary Health</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_temp_health">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_temp_health">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>70</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_damage">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Damage</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_damage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_damage">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>220</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_heal">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="labe_heal">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>250</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Heal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_heal">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>250</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_bonus_log">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>125</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16777215</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_bonus">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>125</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bonus Health</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_current">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Current Health</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_value">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>180</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16777215</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_raw">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Raw Health</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_raw_log">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>110</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16777215</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_temp">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>140</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Temporary Health</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_log">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>140</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16777215</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
